README.md

```diff
@@ -20,56 +20,54 @@ Download publish settings file from https://manage.windowsazure.com/publishsetti
 package main
 
 import (
-    "fmt"
+	"encoding/base64"
+	"fmt"
 
-    "github.com/Azure/azure-sdk-for-go/management"
-    "github.com/Azure/azure-sdk-for-go/management/hostedservice"
-    "github.com/Azure/azure-sdk-for-go/management/virtualmachine"
-    "github.com/Azure/azure-sdk-for-go/management/vmutils"
```

Comment:
Sorry for being unclear...
What I'm saying is that this code does not work if you copy it into a .go file and `go run` it.
The funcs ConfigureDeploymentFromPlatformImage, ConfigureForLinux and ConfigureWithPublicSSH are in the vmutils package. The new readme code doesn't import vmutils (because you copy-pasted code *from* the vmutils package).

BTW, I think you pointed me at the same thing last time I updated this example in the README... :smile: